SampleUsages/TestSamples/AzureQueueShortRunTests.json

```diff
@@ -0,0 +1,28 @@
+﻿[
+  {
+    "functionName": "Queue-NodeJs-CPUIntensive",
+    "platform": "Azure",
+    "inputPath": "..\\queueitems-static-64.config",
```

Comment:
Lets start adding these input files to the repo so that runs can be simple to repeat.